test/test_foreach.py

```diff
@@ -147,9 +147,9 @@ def test_parity(self, device, dtype, op, noncontiguous, inplace):
         else:
             func, ref, _, _ = self._get_funcs(op)
         for sample in op.sample_inputs(device, dtype, noncontiguous=noncontiguous):
-            kwargs = sample.kwargs
-            disable_fastpath = kwargs.pop("disable_fastpath")
-            expect_fastpath = not (noncontiguous or disable_fastpath)
+            ref_kwargs = sample.kwargs
+            kwargs = ref_kwargs.copy()
```

Comment:
Now all this is not necessary, right?

Reply:
It is still necessary because of `"values"` kwarg which I still don't understand how it's used. I want to clean that up in a separate PR, but I don't understand its usage.